src/client/game/game-reducer.js

```diff
@@ -1,23 +1,58 @@
-import { START_GAME, LEAVE_GAME, SET_LEADER } from './game-actions';
+
+import { START_GAME, LEAVE_GAME, SET_LEADER, GET_CONSONANT,
```

Comment:
This list is getting quite long I guess, so it could be changed to `import * as actionTypes from './game-actions'`. Then use `actionTypes.START_GAME` in the reducer and so on